client/js/components/ImageCarousel/styles.js

```diff
@@ -9,6 +9,10 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     position: 'relative',
   },
+  // itemImageContainer: {
+  //   height: 230,
+  //   width: '100%',
+  // },
   imageContainer: {
     width: '100%',
     height: 430,
```

Comment:
Use %s here if it is feasible.

Reply:
I can't change this to percentage the whole carousel changed